src/components/add-comment/add-comment.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import * as api from '../../utils/api';
+import './add-comment.css';
+ 
+const AddComment = () => {
+
+    const {id} = useParams()
+    
+    const [addCommentUsername, setAddCommentUsername] = useState("")
+    const [addCommentBody, setAddCommentBody] = useState("")
+    const [addComment, setAddComment] = useState("")
+ 
+    const handleSubmit = (event) => {
+        const commentToAdd = {
+            username: addCommentUsername,
+            body: addCommentBody
+        };
+
+        event.preventDefault();
+
+        api.postComment(id, commentToAdd)
+        .then((res) => {
+            setAddComment((currComment) => [...currComment, res.data.comment])
+        }).catch((err) => {
+            console.log(err)

```

Comment:
I'd either remove this log, or create some error state/message for when the post doesnt work.